app/audit/models/ChangedLandlineNumberAuditModel.scala

```diff
@@ -19,30 +19,26 @@ package audit.models
 import play.api.libs.json.{JsValue, Json, Writes}
 import utils.JsonObjectSugar
 
-case class ChangedPhoneNumbersAuditModel(currentLandlineNumber: Option[String],
-                                         currentMobileNumber: Option[String],
-                                         requestedLandlineNumber: Option[String],
-                                         requestedMobileNumber: Option[String],
-                                         vrn: String,
-                                         isAgent: Boolean,
-                                         arn: Option[String]) extends ExtendedAuditModel {
+case class ChangedLandlineNumberAuditModel(currentLandlineNumber: Option[String],
+                                           requestedLandlineNumber: Option[String],
```

Comment:
`requestedLandlineNumber` should be a regular `String` and not optional. On the Confluence page we've stated that if they remove their number then we audit it as a blank value but do not omit the field.